BootloaderCommonPkg/Library/IppCryptoLib/rsa_verify.c

```diff
@@ -91,7 +91,12 @@ int VerifyRsaSignature (const void *hash, const void *rsa_n,  const void *rsa_e,
   }
 
   signature_verified = 0;
-  err = ippsRSAVerifyHash_PKCS1v15_rmf((const Ipp8u *)hash, (Ipp8u *)sig, &signature_verified, rsa_key_s, ippsHashMethod_SHA256 (), scratch_buf);
+  if (hash_type == HASH_TYPE_SHA256){
+    err = ippsRSAVerifyHash_PKCS1v15_rmf((const Ipp8u *)hash, (Ipp8u *)sig, &signature_verified, rsa_key_s, ippsHashMethod_SHA256(), scratch_buf);

```

Comment:
can you assign the result of  ippsHashMethod_SHA256()/384() to a variable
and then call the function.

Reply:
Will do.